app/src/main/AndroidManifest.xml

```diff
@@ -89,7 +89,7 @@
     <activity android:name=".testing.BindableAdapterTestActivity" />
     <activity android:name=".testing.ConceptCardFragmentTestActivity" />
     <activity android:name=".testing.ContentCardTestActivity" />
-    <activity android:name=".testing.ContinuePlayingFragmentTestActivity" />
+    <activity android:name=".testing.RecentlyPlayedFragmentTestActivity" />
```

Comment:
Alphabetical ordering of this activity will change

Reply:
Done